crates/uv/tests/it/common/mod.rs

```diff
@@ -240,17 +240,11 @@ impl TestContext {
     #[must_use]
```

Comment:
This is extra clean-up, I could split into a separate change but it's not particularly important.